mkdocs.yml

```
@@ -105,6 +105,8 @@ nav:
       - debug: rpc/debug.md
   - configuration.md
   - Integration options: integration-options.md
+  - Guides:
+      - secure boot: guides/secure-boot.md
```

```suggestion
      - Secure boot: guides/secure-boot.md
```
Consistency with the capitalization style used in the rest of the nav text.